sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubClient/EventHubClientLiveTests.cs

```diff
@@ -215,5 +216,188 @@ await using (var client = new EventHubClient(connectionString))
+        /// </summary>
+        ///
+        [Test]
+        public async Task ClientCanRetrievePartitionPropertiesWithAmqpWebSocketsConnection()
+        {
+            var partitionCount = 4;
+
+            await using (var scope = await EventHubScope.CreateAsync(partitionCount))
+            {
+                var clientOptions = new EventHubClientOptions { TransportType = TransportType.AmqpWebSockets };
+                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
+                var connectionProperties = ConnectionStringParser.Parse(connectionString);
+
+                var credential = new SharedAccessSignatureCredential
+                (
+                    new SharedAccessSignature
+                    (
+                        $"{ clientOptions.TransportType.GetUriScheme() }://{ connectionProperties.Endpoint.Host }/{ connectionProperties.EventHubPath }".ToLowerInvariant(),
+                        connectionProperties.SharedAccessKeyName,
+                        connectionProperties.SharedAccessKey,
+                        TimeSpan.FromHours(4)
+                    )
+                );
+
+                await using (var client = new EventHubClient(connectionProperties.Endpoint.Host, connectionProperties.EventHubPath, credential, clientOptions))
+                {
+                    var cancellation = new CancellationTokenSource(TimeSpan.FromSeconds(20));
+                    var properties = await client.GetPropertiesAsync();
+                    var partition = properties.PartitionIds.First();
+                    var partitionProperties = await client.GetPartitionPropertiesAsync(partition, cancellation.Token);
+
+                    Assert.That(partitionProperties, Is.Not.Null, "A set of partition properties should have been returned.");
+                    Assert.That(partitionProperties.Id, Is.EqualTo(partition), "The partition identifier should match.");
+                    Assert.That(partitionProperties.EventHubPath, Is.EqualTo(connectionProperties.EventHubPath).Using((IEqualityComparer<string>)StringComparer.InvariantCultureIgnoreCase), "The Event Hub path should match.");
+                    Assert.That(partitionProperties.BeginningSequenceNumber, Is.Not.EqualTo(default(Int64)), "The beginning sequence number should have been populated.");
+                    Assert.That(partitionProperties.LastEnqueuedSequenceNumber, Is.Not.EqualTo(default(Int64)), "The last sequance number should have been populated.");
+                    Assert.That(partitionProperties.LastEnqueuedOffset, Is.Not.Null.Or.Empty, "The last offset should have been populated.");
+                }
+            }
+        }
+
+        /// <summary>
+        ///   Verifies that the <see cref="EventHubClient" /> is able to
+        ///   connect to the Event Hubs service.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ClientCanSendWithAmqpWebSocketsConnection()
```

Comment:
The client is not sending or receiving;  the client is simply creating the sender/receiver.   Can we please move these to the `EventHubProducerLiveTests` and `EventHubConsumerLiveTests` classes respectively?

Reply:
I'd also recommend the `[TestCase]` approach for varying the transport types for these.

Reply:
Alright. I'll do that in the other PRs.